content/en/docs/exceptions/_index.md

```diff
@@ -410,3 +410,238 @@ spec:
     - sleep
     - infinity
 ```
+
+## PolicyExceptions with CEL Expressions
+
+Since Kyverno 1.14.0, **PolicyExceptions** support **CEL expressions** to selectively skip policy enforcement for policy types like `ValidatingPolicy` and `ImageValidatingPolicy` in both **admission** and **background** modes.
```

Comment:
```suggestion
Since Kyverno 1.14.0, **PolicyExceptions**, introduced in the new group `policies.kyverno.io`, support **CEL expressions** to selectively skip policy enforcement for policy types like `ValidatingPolicy` and `ImageValidatingPolicy` in both **admission** and **background** modes.
```